general/OKRs/README.md

```diff
@@ -124,22 +124,22 @@ Note: the fewer key results the better, the sample ones are just to illustrate w
 
 ### Engineering:
 
-* Collect, store and process with the engine 10s of millions of repositories using the pipeline in Paris
-  * Key Result 1
-  * Key Result 2
-  * Key Result 3
+* Collect, store and process 10s of millions of repositories using the pipeline in Paris
+  * Improve the borges performance
+  * Scale our infrastructure to support this
+  * Develop a system to keep this updated
 * Launch a production level engine
-  * Key Result 1
-  * Key Result 2
-  * Key Result 3
+  * Improve the performance (using indexes or not)
+  * Support large scale deployments - 10s of millions of repositories
+  * Improve the error reporting
 * Significantly improve the engineering team workflow
-  * Key Result 1
-  * Key Result 2
-  * Key Result 3
+  * Deliver products on time, without commit other teams.
+  * Improve the inter-team workflows.
+  * Improve the communication at all levels.
 * Significantly improving our documentation
-  * Key Result 1
-  * Key Result 2
-  * Key Result 3
+  * Improve engine documentation, with tutorials and examples.
```

Comment:
e,g. A new user without prior knowledge is able to run the engine and complete the tutorials and examples